spec/models/setting_spec.rb

```diff
@@ -169,4 +169,40 @@
     end
   end
 
+  describe ".clear_cache!" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.